designs/Verify-Verbosity.md

```diff
@@ -0,0 +1,306 @@
+
+​                                     | `q[uiet]` | `m[inimal]` | `n[ormal]` | `d[etailed]` | `diag[nostic]`
+------------------------------------- | --------- | ----------- | ---------- | ----------- | --------------
+Errors                                | ✔️        | ✔️         | ✔️         | ✔️         | ✔️   
+Warnings                              | ✔️        | ✔️         | ✔️         | ✔️         | ✔️   
+High-importance Messages              | ❌        | ✔️         | ✔️         | ✔️         | ✔️   
+Normal-importance Messages            | ❌        | ❌         | ✔️         | ✔️         | ✔️   
+Low-importance Messages               | ❌        | ❌         | ❌         | ✔️         | ✔️   
+Additional MSBuild-engine information | ❌        | ❌         | ❌         | ❌         | ✔️   
+
+In other words, you can think of the following log concepts per row as equivalent.
+
+NuGet log level    | MSBuild verbosity switch | MSBuild message type
+------------------ | ------------------------ | --------------------
+`Error`, `Warning` | `q[uiet]`                | Errors and Warnings
+`Minimal`          | `m[inimal]`              | High-importance Messages
+`Information`      | `n[ormal]`               | Normal-importance Messages
+`Verbose`, `Debug` | `d[etailed]`             | Low-importance Messages
+​                   | `diag[nostic]`           | Additional MSBuild-engine information
+
+### Example output in success scenarios
+
+This is output of the invocation of `dotnet nuget verify nuget.common.5.9.0-preview.2.nupkg`
+
+#### Default verbosity | `--verbosity mimimal`
+<details>
+<summary>output</summary>
+
+```
+Verifying NuGet.Common.5.9.0-preview.2
+
+Signature type: Author
+  Subject Name: CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US
+  SHA256 hash: 3F9001EA83C560D712C24CF213C3D312CB3BFF51EE89435D3430BD06B5D0EECE
+  Valid from: 2/25/2018 4:00:00 PM to 1/27/2021 4:00:00 AM
+Timestamp: 11/17/2020 7:23:10 AM
+  Subject Name: CN=Symantec SHA256 TimeStamping Signer - G3, OU=Symantec Trust Network, O=Symantec Corporation, C=US
+  SHA256 hash: C474CE76007D02394E0DA5E4DE7C14C680F9E282013CFEF653EF5DB71FDF61F8
+  Valid from: 12/22/2017 4:00:00 PM to 3/22/2029 4:59:59 PM
+
+Signature type: Repository
+nuget-v3-service-index-url: https://api.nuget.org/v3/index.json
+nuget-package-owners: Microsoft, nuget
+  Subject Name: CN=NuGet.org Repository by Microsoft, O=NuGet.org Repository by Microsoft, L=Redmond, S=Washington, C=US
+  SHA256 hash: 0E5F38F57DC1BCC806D8494F4F90FBCEDD988B46760709CBEEC6F4219AA6157D
+  Valid from: 4/9/2018 5:00:00 PM to 4/14/2021 5:00:00 AM
+Timestamp: 12/9/2020 2:32:12 PM
+  Subject Name: CN=Symantec SHA256 TimeStamping Signer - G3, OU=Symantec Trust Network, O=Symantec Corporation, C=US
+  SHA256 hash: C474CE76007D02394E0DA5E4DE7C14C680F9E282013CFEF653EF5DB71FDF61F8
+  Valid from: 12/22/2017 4:00:00 PM to 3/22/2029 4:59:59 PM
```

Comment:
Do we need to show the trusted timestamp at the default logging level? Can we replace `nuget-v3-service-index-url:` with `Service index:` and `nuget-package-owners` with `Owners:`? Should we change `Subject Name` to `Subject name` to match the capitalization of the other fields?

Reply:
> Can we replace nuget-v3-service-index-url: with Service index: and nuget-package-owners with Owners:? Should we change Subject Name to Subject name to match the capitalization of the other fields?

Fixed in the latest commit.

Reply:
> Do we need to show the trusted timestamp at the default logging level?

As per @dtivel comment here https://github.com/NuGet/NuGet.Client/pull/3774#issuecomment-740216665, I think we can remove `timestamp` certificate details from `minimum` verbosity setting.